core/capabilities/remote/executable/server.go

```diff
@@ -203,6 +203,11 @@ func (r *server) getMessageHash(msg *types.MessageBody) ([32]byte, error) {
 		return [32]byte{}, fmt.Errorf("failed to unmarshal capability request: %w", err)
 	}
 
+	// default to excluding StepDependency
+	if len(r.config.RequestHashExcludedAttributes) == 0 {
```

Comment:
@cedric-cordenier one question: should we _always_ ignore `StepDependency`? As in instead of using it as a default, instead append it to existing RequestHashExcludedAttributes

Reply:
I think it's fine to use it as a default -- if someone explicitly overrides they should put in the right values